### PR TITLE
Added operators.json for Swift

### LINKED ISSUE #311: Add Swift Logic/Math Operators
# Description

Logic/Math Operators for Swift aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right Swift directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of Swift
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by Rutujabadve**
## What GitHub issue does this PR apply to?

Resolves #311

## Reviewer Steps

- [ ] Review commits
- [ ] (If needed) Approve first run of workflow
- [ ] (If needed) Activate Heroku Review app
- [ ] Verify any changes in Deploy Preview
- [ ] Approve and/or merge changes

### CODE REVIEW

File: web/thesauruses/swift/operators.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    },
+    "increment": {
+      "code": "++"
+    },
+    "decrement": {
+      "code": "--"
+    },
+    "exponential": {
+      "not-implemented": "true",
+      "code": ""
```
Comment (geekygirlsarah) [inline web/thesauruses/swift/operators.json:97]: For the lines that are marked `not-implemented`, could you remove the code blocks? It'll help keep the files cleaner and maybe lead to less confusion.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]

**Comment by geekygirlsarah** [comment]
If you make that little change, I think this can be merged in!

**Comment by Rutujabadve** [comment]
I have opened another PR for the requested changes

**Comment by geekygirlsarah** [comment]
Wait, did you make two PRs for the same issue? I'm not sure how I didn't notice this, but also curious how it happened.

**Comment by geekygirlsarah** [comment]
Wait, did you make two PRs for the same issue? I'm not sure how I didn't notice this.

Since this is a duplicate, and the merge conflict is for changes that undo changes in the other one, I'm going to close this PR. (You got Hacktoberfest credit for the other one at least.)

**Comment by Rutujabadve** [comment]
Actually I an new to open source I dint know you could reopen a PR. I was going to close this one.Sorry for inconvinence.

**Comment by geekygirlsarah** [comment]
Oh that's alright! I'm here for helping newer experience people and newer open source contributors learn!

**Pull request closed without merging**